src/main/webapp/WEB-INF/index.yaml

```diff
@@ -16,5 +16,6 @@ indexes:
 - kind: Message
   properties:
   - name: user
+  - name: recipient
```

Comment:
Is this related to images?

Reply:
Not really, but that was a modification to the familiarization projects (Step 7: Update the Datastore index) which was recently added